page-id.ts

```diff
@@ -15,8 +15,8 @@ export enum PageId {
   // thirdweb.com/auth
   AuthenticationLanding = "auth-landing",
 
-  // thirdweb.com/release
-  ReleaseLanding = "release-landing",
```

Comment:
I'm not sure I would change the pageId since they would break posthog history but I defer to @jnsdls on this one